lib/sigh/resign.rb

```diff
@@ -52,9 +56,9 @@ def resign(ipa, signing_identity, provisioning_profile)
     def get_inputs(options, args)
       ipa = args.first || find_ipa || ask('Path to ipa file: ')
       signing_identity = options.signing_identity || ask_for_signing_identity
-      provisioning_profile = options.provisioning_profile || find_provisioning_profile || ask('Path to provisioning file: ')
+      provisioning_profiles = options.provisioning_profile || find_provisioning_profile || ask('Path to provisioning file: ')
```

Comment:
Line is too long. [125/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.